src/CabanaPD_Particles.hpp

```diff
@@ -294,11 +298,23 @@ class Particles<DeviceType, PMB, Dimension>
 
     auto sliceRefPosition()
     {
```

Comment:
@streeve : Since we are using "sliceCurrentPosition" for the current positions, we could use "sliceReferencePosition" for the reference positions, for consistency.

Reply:
Done